Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# gem 'sqlite3'
+gem 'pg', '~> 0.20.0'
```

Comment:
I'm gonna agree with this one. Unless there's some dependency reason we *have* to use PGSQL now, I'd rather either stick with default SQLite or use MySQL.